examples/src/keyring/aws_kms/custom_kms_client_config.py

```diff
@@ -0,0 +1,88 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+By default, the KMS keyring will use the default configurations
+for all KMS clients and will use the default discoverable credentials.
```

Comment:
```suggestion
for all KMS clients and uses the default discoverable credentials.
```
Will readers understand "discoverable credentials?"

Reply:
I'll go through and sprinkle in a link to the docs on that.